src/test/java/ca/corefacility/bioinformatics/irida/service/analysis/workspace/galaxy/impl/integration/AnalysisCollectionServiceGalaxyIT.java

```diff
@@ -121,6 +136,17 @@
 	private static final String FORWARD_NAME = "forward";
 	private static final String REVERSE_NAME = "reverse";
 
+	private SingleEndSequenceFile singleEndFile;
+	private SequenceFile sequenceFile;
+	private UUID workflowId = UUID.randomUUID();
+	private ReferenceFile referenceFile;
+
+	private static final String analysisId = "1";
+
+	private final String analysisName = "analysis 1";
+
+	private User submitter1;
```

Comment:
Much of these new variables are unused now.

Reply:
Removed in 50d4f4fd7d2a884fc1dc8abc61771bc106536aad